extensions/amp-analytics/0.1/requests.js

```diff
@@ -84,6 +87,12 @@ export class RequestHandler {
     /** @private {?Promise<string>} */
     this.baseUrlTemplatePromise_ = null;
 
+    /** @private {?Promise<string>} */
+    this.requestBodyPromise_ = null;
+
+    /** @private {?Promise<string>} */
+    this.requestBodyTemplatePromise_ = null;
```

Comment:
seems this can be a local variable?

Reply:
I followed the pattern used for the baseUrlPromise and baseUrlTemplatePromise for consistency.